src/http-api/resources/files.js

```diff
@@ -87,25 +89,27 @@ exports.get = {
         stream,
         pull.asyncMap((file, cb) => {
           const header = {name: file.path}
-
           if (!file.content) {
             header.type = 'directory'
             pack.entry(header)
             cb()
           } else {
             header.size = file.size
-            toStream.source(file.content)
-              .pipe(pack.entry(header, cb))
+            const packStream = pack.entry(header, cb)
+            if (!packStream) {
+              // this happens if the request is aborted
+              // we just skip things then
+              log('other side hung up')
+              return cb()
+            }
```

Comment:
the `const` has nothing to do with it, `pack.entry` can return `undefined` if the packStream was errored or canceled before.